docs/encryption.md

```diff
@@ -0,0 +1,118 @@
+
+# Example End User Usage
+
+We have added 3 commands in the [`ctr`](https://github.com/containerd/containerd/tree/master/cmd/ctr) client under the image module. They are `ctr image encrypt/decrypt/layerinfo`.
+
+## Encrypt
+
+The following command performs an encryption of the image `docker.io/library/alpine:latest` to an encrypted image with the tag `docker.io/library/alpine:enc`.
+The encryption is done for two recipients with the public key of `/tmp/tmp.AGrSDkaSad/mypubkey.pem` (jwe) and `/tmp/tmp.AGrSDkaSad/clientcert.pem` (pkcs7).
+The option `--layer -1` specifies the layer filter for encryption, -1 indicating the top-most layer should be encrypted.
+
+```
+$ ctr images encrypt \
+    --recipient /tmp/tmp.AGrSDkaSad/mypubkey.pem \
+    --recipient /tmp/tmp.AGrSDkaSad/clientcert.pem \
+    --layer -1 \
+    docker.io/library/alpine:latest docker.io/library/alpine:enc
+
+Encrypting docker.io/library/alpine:latest to docker.io/library/alpine:enc
+```
+
+## Layerinfo
+
+The layerinfo command provides information about the encryption status of an image. In the following command, we use it to inspect the encryption metadata.
+
+```
+$ ctr images layerinfo docker.io/library/alpine:enc
+   #                                                                    DIGEST         PLATFORM      SIZE   ENCRYPTION   RECIPIENTS
+   0   sha256:3427d6934e7749d556be6881a17265c9817abc6447df80a09c8eecc465c5bfb3      linux/amd64   2206947
+   0   sha256:d9a094b6b49fc760501d44ae96f19284e86db0a51b979756ca8a0df4a2746c79     linux/arm/v6   2146469
+   1   sha256:ef87d8b3048d8f1f7af7605328f63aab078a1433116dc15738989551184d7a87     linux/arm/v6       191   jwe,pkcs7    [jwe], [pkcs7]
+   0   sha256:4b0872dff46806a4037c5f158d1d17d5252c9e1f421b7c61445f1a64f6a853a8   linux/arm64/v8   2099778
+   1   sha256:fe022206e6848082f9c1d6e69974157af70ad56bf8698d89e1641d4598bf8ce9   linux/arm64/v8       192   jwe,pkcs7    [jwe], [pkcs7]
+   0   sha256:d1fceb26d4a2dc1f30d05fd0f9567edb5997d504f044ad6486aecc3d5aaa9b4e        linux/386   2271476
+   1   sha256:383a3d4c6789667dbfb6b3742492c4a925315e750f99a5d664ff72f2bb0ae659        linux/386       191   jwe,pkcs7    [jwe], [pkcs7]
+   0   sha256:8aea19b10fd75004ab8fd2d02df719c06528ad3539e686a2d26c933d53f25675    linux/ppc64le   2195242
+   1   sha256:965a60ab5513a2eee33f4d960b63ee347215eb31d06a4ed61f6d90d209462d76    linux/ppc64le       193   jwe,pkcs7    [jwe], [pkcs7]
+   0   sha256:783541963cb4e52173193fe947bb7a7f7e5a6657a4cbbb6b8b077bbee7255605      linux/s390x   2307762
+   1   sha256:69d3260b3f5430ade9a3ee0f1b71a32a8e4ef268552beeae29930a8795dc54bf      linux/s390x       192   jwe,pkcs7    [jwe], [pkcs7]
+```
+
+## Decrypt
+
+The following command performs an decryption of the encrypted image `docker.io/library/alpine:enc` to the image tag `docker.io/library/alpine:latest`.
```

Comment:
This sentence doesn't match the example below.

Reply:
Fixed! 71a6a8d752dbc81ac1bc025d1cbf8411e98218b7